config/settings.yml

```diff
@@ -19,3 +19,5 @@ plugins:
   show_filter_by_solved_status:
     default: false
     client: true
+  notify_op_on_staff_accept:
```

Comment:
I feel like the name should include `solved` like the other settings. Maybe `notify_on_staff_accept_solved`

Reply:
Agreed here. Would be hard to search then if without.